### PR TITLE
Upgrade version for 1.14.9

### PR DESCRIPTION
**Pull request by abenbachir**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ If you are an Azure customer, we have an Azure VM extension that allows you to e
 
 ## [Full installation guide](https://docs.microsoft.com/azure/azure-monitor/platform/log-analytics-agent)
 
-## [Download Latest OMS Agent for Linux (64-bit)](https://github.com/microsoft/OMS-Agent-for-Linux/releases/download/OMSAgent_v1.13.40-0/omsagent-1.13.40-0.universal.x64.sh)
+## [Download Latest OMS Agent for Linux (64-bit)](https://github.com/microsoft/OMS-Agent-for-Linux/releases/download/OMSAgent_v1.14.1-0/omsagent-1.14.9-0.universal.x64.sh)
```
Comment (hestolz) [inline README.md:34]: how come the release tag is `1.14.1-0`?

File: docs/OMS-Agent-for-Linux.md [hunk 2]
```diff
@@ -17,10 +17,10 @@ The Log Analytics agent for Linux comprises multiple packages. The release file
 
 **Package** | **Version** | **Description**
 ----------- | ----------- | --------------
-omsagent | 1.13.40-0 | The Operations Management Suite Agent for Linux
-omsconfig | 1.1.1-930 | Configuration agent for the OMS Agent
+omsagent | 1.14.9-0 | The Operations Management Suite Agent for Linux
+omsconfig | 1.1.1-931 | Configuration agent for the OMS Agent
 omi | 1.6.8-1 | Open Management Infrastructure (OMI) -- a lightweight CIM Server. *Note that OMI requires root access to run a cron job necessary for the functioning of the service*
-scx | 1.6.4-7 | OMI CIM Providers for operating system performance metrics
+scx | 1.648-1 | OMI CIM Providers for operating system performance metrics
```
Comment (hestolz) [inline docs/OMS-Agent-for-Linux.md:23]: isn't it `1.6.8-1`?

File: installer/scripts/onboard_agent.sh [hunk 3]
```diff
@@ -6,10 +6,10 @@
 
 
 # Values to be updated upon each new release
-GITHUB_RELEASE_X64="https://github.com/microsoft/OMS-Agent-for-Linux/releases/download/OMSAgent_v1.13.40-0/"
+GITHUB_RELEASE_X64="https://github.com/microsoft/OMS-Agent-for-Linux/releases/download/OMSAgent_v1.14.1-0/"
```
Comment (hestolz) [inline installer/scripts/onboard_agent.sh:9]: again, why tag 1.14.1?

File: README.md (hunk 1 above)
Comment (abenbachir) [reply README.md]: fixed it

File: installer/scripts/onboard_agent.sh (hunk 3 above)
Comment (abenbachir) [reply installer/scripts/onboard_agent.sh]: initially I started the release as 1.14.1, and it becomes 1.14.9, I fixed it.

**Review by hestolz — APPROVED** [review]

File: docs/OMS-Agent-for-Linux.md (hunk 2 above)
Comment (abenbachir) [reply docs/OMS-Agent-for-Linux.md]: done

File: installer/scripts/onboard_agent.sh (hunk 3 above)
Comment (wUEZRs) [reply installer/scripts/onboard_agent.sh]: When we run wget https://raw.githubusercontent.com/Microsoft/OMS-Agent-for-Linux/master/installer/scripts/onboard_agent.sh && sh onboard_agent.sh -w <YOUR WORKSPACE ID> -s <YOUR WORKSPACE PRIMARY KEY> on our Linux machines it installs 1.13.40. Isn't expected behavior that it upgrades to 1.14.x?

**Pull request merged**